.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,97 @@
+        service: [ 'user-service', 'gateway' ]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup-maven
+      - name: Compile and install
+        run: |
+          mvn clean install -DskipTests=true
+      - name: Create .env file
+        run: |
+          echo "POSTGRES_PASSWORD=${{ secrets.POSTGRES_TEST_PASSWORD }}" > .env
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)